src/pages/ClaimsPage/utils.ts

```diff
@@ -33,9 +33,11 @@ export function formatClaimAmount(claims: QueryClaim[]): string {
   const totalAmount = claims.reduce((accumulator, curr) => {
     return accumulator + (curr.unwrappedNewAmount || 0);
   }, 0);
-  return `${parseFloat(totalAmount.toString()).toFixed(2)} ${
-    claims[0].distribution.vault.symbol
-  }`;
+  return `${formatAmount(totalAmount)} ${claims[0].distribution.vault.symbol}`;
+}
+
+export function formatAmount(amount: number): string {
```

Comment:
Can we just use lodash.round for this?

Reply:
No, for numbers without decimals `round` doesn't include the decimals.

![image](https://user-images.githubusercontent.com/83605543/183991734-08c528bd-0258-4be2-80f7-8bf62d3c9f5a.png)